easybuild/tools/utilities.py

```diff
@@ -132,9 +133,15 @@ def import_available_modules(namespace):
     :param namespace: The namespace to import modules from.
     """
     modules = []
-    for path in sys.path:
 
-        cand_modpath_glob = os.path.sep.join([path] + namespace.split('.') + ['*.py'])
+    try:
+        mod = importlib.import_module(namespace)
+    except ImportError as err:
+        raise EasyBuildError("import_available_modules: Failed to import %s: %s", namespace, err)
```

Comment:
This changes the semantics of `import_available_modules`, since no hard error was being raised before when the specified namespace didn't exist?